src/Sulu/Bundle/AdminBundle/Tests/Unit/FormMetadata/dummy-forms/dummy-form.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" ?>
+<form xmlns="http://schemas.sulu.io/template/template"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/form-1.0.xsd"
+>
+    <key>dummy-form</key>
```

Comment:
Why is this form necessary?

Reply:
because I can't mock the `Finder`, I need to provide a real directory with a real xml file

Reply:
theoretically it could also be a completely empty xml file, but that would be very confusing IMO